doc/usage/restructuredtext/roles.rst

```diff
@@ -249,6 +249,34 @@ There is also an :rst:role:`index` role to generate index entries.
 
 The following roles generate external links:
 
+.. rst:role:: cve
+
+   A reference to a `Common Vulnerabilities and Exposures`_ record.
+   This generates appropriate index entries.
+   The text "CVE *number*\ " is generated;
+   with a link to an online copy of the specified CVE.
+   You can link to a specific section by using ``:cve:`number#anchor```.
```

Comment:
Don't we need something like:

```rst
``:cve:\`number#anchor\```
```

Reply:
This works: https://sphinx--11781.org.readthedocs.build/en/11781/usage/restructuredtext/roles.html#role-cve

https://sphinx--11781.org.readthedocs.build/en/11781/usage/restructuredtext/roles.html#role-cwe

Reply:
Sphinx is smarter than GitHub for that! (too many times when I needed to use HTML hacks on GH for this kind of things...)